storage/bootstrap/provider_options.go

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) 2018 Uber Technologies, Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+package bootstrap
+
+const (
+	// defaultCacheSeriesMetadata declares that by default bootstrap providers should
+	// cache series metadata between runs.
+	defaultCacheSeriesMetadata = true
+)
+
+type processProviderOptions struct {
+	cacheSeriesMetadata bool
+}
+
+// NewProcessProviderOptions creates new bootstrap run options
+func NewProcessProviderOptions() ProcessProviderOptions {
```

Comment:
nit: Maybe we should call these ProcessOptions?  They just so happen to be passed to the "Provider" for the "Process". (We're kind of using "Provider" to mean "Factory" here, but we don't want to sound Java-y, it's a little silly perhaps).